src/User/ProfilePage.test.jsx

```diff
@@ -40,8 +40,45 @@ describe("ProfilePage", () => {
 
   const mockManagerName = "Joe Bloggs";
 
-  it("renders current user's profile information with manager", async () => {
-    expect.assertions(7);
+  const mockEmployees = [
+    {
+      id: 12,
+      username: "Employee1",
+      firstName: "Employee",
+      lastName: "One",
+      email: "Employee.One@live.co.uk",
+      occupation: "Test Employee",
+      managerId: 11
+    },
+    {
+      id: 13,
+      username: "Employee2",
+      firstName: "Employee",
+      lastName: "Two",
+      email: "Employee.Two@live.co.uk",
+      occupation: "Test Employee",
+      managerId: 11
+    },
+    {
+      id: 14,
+      username: "Employee3",
+      firstName: "Employee",
+      lastName: "Three",
+      email: "Employee.Three@live.co.uk",
+      occupation: "Test Employee",
+      managerId: 11
+    }
+  ];
+
+  const employeeName = (employeeList, employeeIndex) => {
+    const employee = employeeList[employeeIndex];
+    return `${employee.firstName} ${employee.lastName}`;
+  };
+
+  const mockNoEmployees = [];
+
+  it("renders current user's profile information with manager and employees", async () => {
```

Comment:
Would probably treat the employees as a seperate unit test

Reply:
Will do!